test/TensorFlowRuntime/sese_loop_canonicalization.swift

```diff
@@ -69,11 +67,9 @@ ControlFlowTests.testAllBackends("sumOfProducts") {
 func sumOfProductsWithBound(_ M : Int32, _ N : Int32, _ Bound : Float) -> Tensor<Float> {
   // Effectively computes natSum(M)*natSum(N) upto Bound
   var sum = Tensor<Float>(0)
-outer_loop:for i in 0..<M {
-    for j in 0..<N {
-      // i + 1 and j + 1 is a workaround for
-      // https://bugs.swift.org/browse/SR-8256
-      let prod = Tensor<Float>(Float((i + 1) * (j + 1)))
+outer_loop:for i in 1...M {
```

Comment:
can we fix/improve the identation here?

Reply:
I believe he meant to move the `for`-loop to a new line below the label.